src/providers/https.ts

```diff
@@ -163,88 +195,44 @@ export class HttpsError extends Error {
    * A standard error code that will be returned to the client. This also
    * determines the HTTP status code of the response, as defined in code.proto.
    */
-  readonly code: FunctionsErrorCode;
+  public readonly code: FunctionsErrorCode;
 
   /**
    * Extra data to be converted to JSON and included in the error response.
    */
-  readonly details?: unknown;
+  public readonly details: unknown;
+
+  /**
+   * A wire format representation of a provided error code.
+   */
+  public readonly httpErrorCode: HttpErrorCode;
```

Comment:
Not this is not `@hidden`. I'm not sure if we're allowed to make this change without consultation with the API council, but I decided to give it a shot as I found it very useful in some of the projects to have this mapping and I ended up duplicating the code. It would be very convenient to have it available.

Reply:
I think that makes sense - we can expose this. In fact this is great timing as we've been doing some docs changes and doing a full pass through to see whether methods/variables should be exposed or not. Let me know if you find others that you would find helpful to expose.

Reply:
Ah, I missed a couple things here. On second pass, I see this is part of the `HttpsError` class, which we cannot change without API review. Let's take this out for now, and I can follow up on this later after discussing with the API council. Apologies for the confusion!